aRchive.py

```diff
@@ -143,13 +151,27 @@ def archive_package_versions(bioc_pack, archive_dir):
             log.debug("Bioc_pack version %s %s" % (bioc_pack, curr_version))
             # Create new directory with version number as "-version" extension
             bioc_pack_name = os.path.split(bioc_pack)[-1]
-            output_directory = os.path.join(archive_dir,
-                                            bioc_pack_name, curr_version)
-            if not os.path.exists(output_directory):
-                log.info("Made new version directory: %s" % output_directory)
+            output_directory = os.path.join(archive_dir)
+            out_tarfile = bioc_pack_name + "_" + curr_version + '.tar.gz'
```

Comment:
might switch to formatting a string `"%s_%s.tar.gz" % (bioc_pack_name, curr_version)`